packages/0x.js/test/utils/web3_wrapper.ts

```diff
@@ -1,6 +1,10 @@
-import { web3Factory } from '@0xproject/dev-utils';
+import { devConstants, web3Factory } from '@0xproject/dev-utils';
 import { Web3Wrapper } from '@0xproject/web3-wrapper';
 import * as Web3 from 'web3';
 
-export const web3 = web3Factory.create();
-export const web3Wrapper = new Web3Wrapper(web3.currentProvider);
+import { constants } from './constants';
+
+const web3 = web3Factory.create({ shouldUseInProcessGanache: true });
+const web3Wrapper = new Web3Wrapper(web3.currentProvider);
+
+export { web3, web3Wrapper };
```

Comment:
I think exporting provider here helps reduce Web3 imports for `web3.currentProvider`, like in `packages/deployer/test/util/provider.ts`